legistar/events.py

```diff
@@ -89,7 +89,10 @@ def events(self, follow_links=True, since=None):
                     yield event, agenda
                     no_events_in_year = False
 
-            if no_events_in_year:  # Bail from scrape if no results returned from year
+            # We search for events for older and older years. If older year doesn't
```

Comment:
@hancush what do you think about this comment, instead?

Reply:
Hm, I see what you mean about the prior version being tricky, but I think the part that really needs explanation is the `year <= current_year`. What about:

> We scrape events in reverse chronological order, starting one year in the future. Stop scraping if there are no events in a given year, unless that year is in the future, because whether events have been scheduled for the future is not a reliable indication of whether any happened in the previous year.

Reply:
sure

Reply:
Cool, updated in 49e9cce.